dev/com.ibm.ws.jsf.2.3_fat/fat/src/com/ibm/ws/jsf23/fat/tests/JSF23CDIInjectionTests.java

```diff
@@ -90,6 +91,7 @@ public static void tearDown() throws Exception {
      *
      */
     @Test
+    @SkipForRepeat(SkipForRepeat.EE10_FEATURES) // looking in sessionmap for session bean
     public void testActionListenerInjection_CDIInjectionTests() throws Exception {
         testActionListenerInjectionByApp("ActionListenerInjection", server);
     }
```

Comment:
At first I didn't quite get it, but now I see. 

https://github.com/OpenLiberty/open-liberty/blob/2fd4a880754c37a988c5ed9ac4f1ea5988e465d6/dev/com.ibm.ws.jsf.2.3_fat/test-applications/CDICommon/src/com/ibm/ws/jsf23/fat/cdi/common/managed/CustomActionListener.java#L66


The Session Scoped Managed Beans was stored in the session, but not for CDI.